modules/primo/primo.module

```diff
@@ -133,7 +133,8 @@ function primo_client() {
       'base_uri' => variable_get('primo_base_url')
     ]),
     variable_get('primo_institution_code'),
-    ip_address()
+    ip_address(),
+    variable_get('primo_location_scopes', [])
```

Comment:
There is currently no administrative interface for settings this variable?

Reply:
Ah, you're right - should I add it? Won't be that much work to implode/explode the array into a text-area

Reply:
I think it would be very appropriate.